pkg/api/addons.go

```diff
@@ -962,6 +1009,13 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 			o.KubernetesConfig.Addons[i].Enabled = to.BoolPtr(true)
 		}
 	}
+
+	// Enable keyvault flexvolume addon during upgrade for 1.16 or greater scenarios, unless explicitly disabled
+	if isUpgrade && common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.16.0") && !o.KubernetesConfig.IsAddonDisabled(common.KeyVaultFlexVolumeAddonName) {
```

Comment:
you should be able to do an `IsAddonEnabled` instead of the double negative

Reply:
thanks, done!